ui/apps/platform/cypress/integration/violations/filteredWorkflowViews.test.ts

```diff
@@ -0,0 +1,87 @@
+import withAuth from '../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../helpers/features';
+import { visitViolations } from './Violations.helpers';
+
+const filteredWorkflowSelectButton = 'button[aria-label="Filtered workflow select"]';
+const getFilteredWorkflowOptionButton = (viewName) =>
+    `div[aria-label="Filtered workflow select options"] button:contains("${viewName}")`;
+
+describe('Violations - Filtered Workflow Views', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_PLATFORM_COMPONENTS')) {
+            this.skip();
+        }
+    });
+
+    it('should render the correct filtered workflow view select options', () => {
```

Comment:
While this could technically qualify as a component test, I felt it was appropriate to test it as an end-to-end (e2e) test alongside the tests listed below.

Reply:
This makes sense. I think making a separate file for a single component level test that runs in a separate CI job, while technically "correct", is probably more indirection than we need for what it would accomplish.

Alternatively, I could see an argument that maybe we don't even need this test at all. The properties under test are all covered by the other tests in this suite, and a violation of this test should be immediately noticeable to anyone working on the feature. (If the properties were determined dynamically, versus being a static part of this component, this would hold less weight though.